apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -1535,6 +2332,106 @@ module.exports = {
+                {
+                  label: "Title",
+                  name: "title",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Description",
+                  name: "description",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Open Graph",
+                  name: "openGraph",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Title",
+                      name: "title",
+                      widget: "string",
+                      required: false,
+                    },
+                    {
+                      label: "Description",
+                      name: "description",
+                      widget: "string",
+                      required: false,
+                    },
+                    {
+                      label: "Images",
+                      name: "images",
+                      widget: "list",
+                      required: false,
+                      fields: [
+                        {
+                          name: "url",
+                          label: "URL",
+                          widget: "string",
+                        },
+                        {
+                          name: "width",
+                          label: "Width",
+                          widget: "string",
+                        },
+                        {
+                          name: "height",
+                          label: "Height",
+                          widget: "string",
+                        },
+                        {
+                          name: "alt",
+                          label: "Alt Text",
+                          widget: "string",
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Twitter",
+                  name: "twitter",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Handle",
+                      name: "handle",
+                      widget: "string",
+                      required: false,
+                      default: "@Code4Africa",
+                    },
+                    {
+                      label: "Site",
+                      name: "site",
+                      widget: "string",
+                      required: false,
+                      default: "@Code4Africa",
+                    },
+                    {
+                      label: "Card Type",
+                      name: "cardType",
+                      widget: "hidden",
+                      default: "summary_large_image",
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        },
```

Comment:
1. Move this to above header. The order should be General, Header and then Footer.
2. Lets update it to 👇🏽 
```suggestion
         {
          name: "general",
          label: "General",
          file: `${APP_DIRECTORY}content/settings/general.md`,
          fields: [
            {
              label: "Site",
              name: "site",
              widget: "object",
              fields: [
                {
                  label: "Title",
                  name: "title",
                  widget: "string",
                  hint: "The name of the site",
                },
                {
                  label: "Description",
                  name: "description",
                  widget: "text",
                  required: false,
                  hint: "Helps with search results and when shared in social media platforms",
                },
              ],
            },
            {
              label: "SEO",
              name: "seo",
              widget: "object",
              fields: [
                {
                  label: "Title Template",
                  name: "title-template",
                  widget: "string",
                  hint: '"pre/suffix that should be included with every page. It replaces %s with your title string e.g. "%s | CfA" template will render "About | CfA" in about page with title "About"',
                },
                {
                  label: "Meta data",
                  name: "meta",
                  widget: "object",
                  hint: "Search engines support",
                  fields: [
                    {
                      label: "Title",
                      name: "title",
                      widget: "string",
                      required: false,
                      hint: "Uses site title if not provided",
                      pattern: ["^.{1,70}$", "Must be up to 70 characters"],
                    },
                    {
                      label: "Description",
                      name: "description",
                      widget: "text",
                      required: false,
                      hint: "Uses site description if not provided",
                      pattern: ["^.{1,150}$", "Must be up to 156 characters"],
                    },
                  ],
                },
                {
                  label: "Open Graph",
                  name: "open-graph",
                  hint: "Facebook, Slack, and other social media platforms",
                  widget: "object",
                  fields: [
                    {
                      label: "Title",
                      name: "title",
                      hint: "Uses site title if not provided",
                      widget: "string",
                      required: false,
                    },
                    {
                      label: "Description",
                      name: "description",
                      widget: "string",
                      hint: "Uses site description if not provided",
                      required: false,
                    },
                    {
                      label: "Images",
                      name: "images",
                      widget: "list",
                      min: 1,
                      max: 1,
                      required: false,
                      fields: [
                        {
                          name: "url",
                          label: "URL",
                          widget: "string",
                        },
                        {
                          name: "width",
                          label: "Width",
                          widget: "number",
                        },
                        {
                          name: "height",
                          label: "Height",
                          widget: "number",
                        },
                        {
                          name: "alt",
                          label: "Alt Text",
                          widget: "string",
                        },
                      ],
                    },
                  ],
                },
                {
                  label: "Twitter",
                  name: "twitter",
                  widget: "object",
                  fields: [
                    {
                      label: "Site",
                      name: "site",
                      hint: "@username of website",
                      widget: "string",
                      required: false,
                      default: "@Code4Africa",
                    },
                    {
                      label: "Card Type",
                      name: "cardType",
                      widget: "hidden",
                      default: "summary_large_image",
                    },
                  ],
                },
              ],
            },
          ],
        },
```